_layouts/menu.html

```diff
@@ -18,12 +18,12 @@
     </button>
     <ul class="submenu">
       <li class="submenu-item">
-        <a class="menu-link" href="https://github.com/tc39/agendas/blob/master/2021/05.md">
+        <a class="menu-link" href="https://github.com/tc39/agendas/blob/main/2022/03.md">
```

Comment:
```suggestion
        <a class="menu-link" href="https://github.com/tc39/agendas/blob/HEAD/2022/03.md">
```
better to avoid hardcoding the default branch name